src/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,16 @@
+import {useState} from 'react';
+
+export const useLocalStorage = (key, initialValue) => {
+    const [storedValue, setStoredValue] = useState(() => {
+        const item = window.localStorage.getItem(key);
+        return item ? JSON.parse(item) : initialValue;
+    });
+
+    const setValue = value => {
+        setStoredValue(value);
+        window.localStorage.setItem(key, JSON.stringify(value));
```

Comment:
Awesome work using local storage. Pop quiz: would storing a user's login information be a good use of local storage? Why or why not?